service/src/main/java/uk/gov/hmcts/reform/fpl/service/CourtService.java

```diff
@@ -36,6 +40,25 @@ public String getCourtEmail(CaseData caseData) {
             .orElse(null);
     }
 
+    public String getPreviousCourtName(CaseData caseData) {
+        if (!caseData.getPastCourtList().isEmpty()) {
+            Court lastCourt = null;
+            if (caseData.getPastCourtList().size() == 1) {
+                lastCourt = caseData.getPastCourtList().iterator().next().getValue();
+            } else {
+                lastCourt = caseData.getPastCourtList().stream()
+                    .map(Element::getValue)
+                    .filter(c -> !Objects.isNull(c.getDateTransferred()))
```

Comment:
will this transferred date be null for all existing courts?

is that why we need the above .size() == 1 check, to handle the first court transfer?